package.json

```diff
@@ -3,31 +3,48 @@
   "version": "0.11.1",
   "description": "Refuge from unsafe JavaScript",
   "license": "MIT",
+  "main": "lib/index",
+  "jsnext:main": "es6/index",
   "repository": {
     "type": "git",
     "url": "git://github.com/sanctuary-js/sanctuary.git"
   },
   "scripts": {
-    "test": "make lint test"
+    "test": "make lint test",
+    "cleanup": "rm -rf es6 lib umd",
+    "build": "npm run cleanup && npm run build-cjs && npm run build-es",
+    "build-es": "BABEL_ENV=es babel ./src -d es6",
+    "build-cjs": "BABEL_ENV=cjs webpack src/index.js lib/index.js",
+    "build-umd": "NODE_ENV=development webpack src/index.js umd/sanctuary.js",
+    "build-min": "NODE_ENV=production webpack -p src/index.js umd/sanctuary.min.js"
   },
   "dependencies": {
     "ramda": "0.22.x",
     "sanctuary-def": "0.6.x"
   },
   "devDependencies": {
+    "babel-cli": "^6.14.0",
+    "babel-core": "^6.13.2",
+    "babel-eslint": "^6.1.2",
+    "babel-loader": "^6.2.5",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-preset-es2015": "^6.13.2",
```

Comment:
Let's use `"1.2.x"` rather than `"^1.2.3"` for consistency with the way other dependencies are declared.